### ISSUE TITLE
E349 on commit (tmux only)

### ISSUE THREAD

**Issue by rene-aguirre** [tpope/vim-fugitive#890]
Keeping in case someone bumps into the same issue, my config:
 * iTerm2
 * tmux, configured as xterm-256 (for proper colorscheme rendering) and "screen-256color" as suggested by tmux documentation.
 * vim only, neovim (nvim) is not affected, and obviously not gvim/macvim
 * Maybe relevant, also using "tmux-plugins/vim-tmux-focus-events" plugin with related tmux configuration.

When trying to commit from fugitive, I was getting an empty screen and a cursor error message with E349 prefix: "no Identifier under cursor".

I found the right setting here: https://superuser.com/questions/401926/how-to-get-shiftarrows-and-ctrlarrows-working-in-vim-in-tmux.

In summary, add this to your vim configuration:

```
if &term =~ '^screen'
    execute "set <xUp>=\e[1;*A"
    execute "set <xDown>=\e[1;*B"
    execute "set <xRight>=\e[1;*C"
    execute "set <xLeft>=\e[1;*D"
	map <Esc>[B <Down>
endif
```
